server/src/main/resources/configuration.yml

```diff
@@ -23,12 +23,41 @@ authentication:
       issuer: "http://localhost:9090/default"
       ignoreExpiration: true
 
-authorizationEnabled: false
+authorizationEnabled: true
 
 authorization:
   superUsers: [ "thanos" ]
-  providerClassName:
-  providerOptions:
+  providerClassName: "com.flipkart.varadhi.auth.DefaultAuthorizationProvider"
```

Comment:
fine for POC, but for prod it might be better to extract these to authz specific file and have a pointer in the main config file.

Reply:
sure will do that